spec/tasks/achievements/import/two_valid_achievements.yml

```diff
@@ -0,0 +1,9 @@
+- title: First
+  side: runner
+  points: 1
+  description: First
+---
+- title: Second
+  side: corp
+  points: 2
+  description: Second
```

Comment:
Small nitpick: would you mind moving these YAML files into `spec/fixtures` (which I think you'll need to create). Ideally spec folders only contain specs, and arbitrary data files are stored in a separate fixtures folder. Good work on the specs though, I always feel better with a robust test suite under my project! :+1: